packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
```

Comment:
I'm not 100% sure of this, but addresses are 42 characters (including `0x`). https://kb.myetherwallet.com/en/getting-started/glossary-of-common-terms/ @bbenligiray can you confirm?

Reply:
`endpointId` is a `bytes32` type so it's 64 hexadecimals